xc/xc7/archs/artix7_100t/tiles/int_l/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+project_ray_tile(
```

Comment:
The INT_L/INT_R tiles are unused, so can be deleted safely, here and in the A50T and A200T directories.

Reply:
Agreed, the tests are running now with these (and the CLK_HROW_* tiles) deleted.